src/inject/generic/core/webviewLoader.js

```diff
@@ -7,7 +7,9 @@ if (process.env.TEST_SPEC) {
 
 if (webview) {
   let once = true;
-  const targetPage = Settings.get('lastPage', 'https://play.google.com/music/listen');
+  const targetPage = Settings.get('savePage', true) ?
+Settings.get('lastPage', 'https://play.google.com/music/listen') :
```

Comment:
As a ternary expression I would expect it to be indented 2 spaces in from the `=` sign on the top line.